templates/widget/tlmswidget.php

```diff
@@ -4,7 +4,7 @@
             <a href="<?php echo esc_url(get_site_url()); ?>/courses/?tlms-course=<?php echo esc_attr($course->id); ?>">
                 <img src="<?php echo esc_attr($course->big_avatar); ?>" alt="<?php echo esc_attr($course->name); ?>"/>
                 <span class="tlms-widget-caption"> <?php echo esc_html($course->name);
-                    echo ($course->course_code) ? "(".esc_html($course->course_code).")" : ''; ?>
+                    echo (esc_html($course->course_code)) ? "(".esc_html($course->course_code).")" : ''; ?>
```

Comment:
Do we actually need it? How about using `empty($course->course_code)`?

Reply:
Yes, agreed. Replaced with `!empty($course->course_code)`